library/common/http/client.h

```diff
@@ -234,6 +241,15 @@ class Client : public Logger::Loggable<Logger::Id::http> {
     Client& parent_;
     // Response details used by the connection manager.
     absl::string_view response_details_;
+    // True if the bridge should operate in explicit flow control mode.
+    //
+    // In this mode only one callback can be sent to the bridge until more is
+    // asked for. When a response is started this will either allow headers or an
+    // error to be sent up. Body, trailers, or further errors will not be sent
+    // until resumeData is called. This, combined with standard Envoy flow control push
```

Comment:
resumeData, or readData?

Reply:
readData was what I landed on. See my comment below.

Reply:
I didn't update client.h and client.cc, just because wanted to err on the side of caution and not cause unnecessary conflicts with any branches @alyssawilk might have to land on top.

Reply:
Just realized you are completely right about there being a missing step here. Updating.